src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: $padding;
+  font-family: Arial, Helvetica, sans-serif;
+
+  @for $i from 28 through 31 {
+    $hide_count: 31 - $i;
+
+    &--month-length-#{$i} div:nth-last-child(-n + #{$hide_count}) {
+      display: none;
```

Comment:
Good job! Please, check 'checklist.md' file p.7, I think you should use visibility: hidden, beacuse display we can't use